common/fee/types/fee.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+import (
+	"fmt"
+)
+
+// Opt is an option for a gas estimator
+type Opt int
+
+const (
+	// OptForceRefetch forces the estimator to bust a cache if necessary
+	OptForceRefetch Opt = iota
```

Comment:
Looks pre-existing and out-of-scope, but did we consider using a bitmask for these options? Seems like a trivial improvement on the surface, but would prevent the method signatures from blowing up with variadics or slices.

Reply:
yeah it's pre-existing, but definitely something to look into in the future
captured the improvement here: https://smartcontract-it.atlassian.net/browse/BCI-1408